README.md

```diff
@@ -1,17 +1,18 @@
-# Translation relay server for brave
+# Translation server for brave
 
-`go-translate` implements a translation relay server for use in brave-core written in Go.
+`go-translate` implements a translation server for use in brave-core written in Go.
+It works on top of brave-hosted [Lingvanex On-premise Translation Server](https://lingvanex.com/translationserver/). It gets the requests from browsers in Chromium format,
+rewrite them to Lingvanex format, process requests and returns the result back to Chromium format.
```

Comment:
rewrites, processes the requests
back in Chromium format

Reply:
```suggestion
rewrites them to Lingvanex format, processes the requests and returns the result back in Chromium format.
```